### DISCUSSION TITLE
Exclude/disable specific components?

### DISCUSSION THREAD

**Discussion by ekafyi** [saadeghi/daisyui#790] [Q&A] [answered]
Hi, is it possible at all to exclude/disable a specific component classname, eg. `.hero`?

Case: Working in an existing project that has components/markup using the same class name in multiple places; prefer not to touch (rename) them. I'm not using daisyui's Hero component. I do use other components from daisyui and do not want to remove the styling entirely.

Thanks!

**Comment by saadeghi** [comment, marked as answer]
It's not possible to exclude a specific component yet. I'm planning to add this feature in the next major version.

For now, you can use `prefix` config to add a prefix to all daisyUI classes to avoid conflicts.
